xcode/Subconscious/Shared/Components/FirstRun/FirstRunView.swift

```diff
@@ -16,17 +16,16 @@ struct FirstRunView: View {
             VStack(spacing: AppTheme.padding * 2) {
                 Spacer()
                 StackedGlowingImage(
-                    image: Image("sub_logo"),
                     width: AppTheme.onboarding.heroIconSize,
                     height: AppTheme.onboarding.heroIconSize
-                )
+                ) {
+                    Image("sub_logo").resizable()
+                }
                 Spacer()
                 VStack(alignment: .leading, spacing: AppTheme.unit3) {
-                    Text("Welcome to the Subconscious Beta.")
-                    
-                    Text("Subconscious is a place to garden thoughts and share with others.")
+                    Text("Subconscious is a place to garden thoughts and share them with others.")
```

Comment:
So much of this view is centered, I feel we should embrace it and center the text as well?

<img src="https://github.com/subconsciousnetwork/subconscious/assets/58421/7a3b71c2-e443-4401-90ae-097aedc4c626" width="500">

Reply:
I centered everything and got this view playing nicely with the keyboard (by hiding the button and shrinking the logo). It _does not_ actually submit the step using the keyboard submit action yet (see https://github.com/subconsciousnetwork/subconscious/issues/620 for why).

We still need to work out how to make the AppUpgradeView align properly with this, with the dynamic sizing it seems tricky. I'd be tempted to open an issue just for that.

<img width="373" alt="Screenshot 2023-05-11 at 11 19 12 am" src="https://github.com/subconsciousnetwork/subconscious/assets/5009316/854e3a3b-84e4-48fc-b408-5b19deb6e9a9">
<img width="372" alt="Screenshot 2023-05-11 at 11 19 05 am" src="https://github.com/subconsciousnetwork/subconscious/assets/5009316/0cb8c3ba-5b38-48de-8a30-2110610cc844">